src/chrome/content/rules/lawinenwarndienst-bayern.de.xml

```diff
@@ -0,0 +1,14 @@
+<ruleset name="Lawinenwarndienst Bayern">
+
+	<target host="lawinenwarndienst-bayern.de" />
+	<target host="www.lawinenwarndienst-bayern.de" />
+
+	<securecookie host=".+" name=".+" />
+
+	<!-- Server not fount on https://lawinenwarndienst-bayern.de/,
+	http://lawinenwarndienst-bayern.de/ redirects to http://www.lawinenwarndienst-bayern.de/ -->
```

Comment:
That's a hack in browsers (see `browser.fixup.alternate.enabled` in Firefox). `lawinenwarndienst-bayern.de` does not exist.
```
$ curl lawinenwarndienst-bayern.de
curl: (6) Could not resolve host: lawinenwarndienst-bayern.de